lib/smart_answer_flows/check-uk-visa/outcomes/outcome_marriage_visa_nat_datv.govspeak.erb

```diff
@@ -5,6 +5,10 @@
 <% content_for :body do %>
   <%= render partial: 'stateless_or_refugee.govspeak.erb', locals: {calculator: calculator} %>
 
+  <% if calculator.passport_country_is_estonia? || calculator.passport_country_is_latvia? %>
```

Comment:
@ikennaokpala I see your point. I would have made sense to split the logic change in the flow from the additional logic in the template.